code/modules/reagents/chemistry/reagents/drug_reagents.dm

```diff
@@ -288,6 +284,12 @@
 	M.adjustStaminaLoss(-5, 0)
 	M.adjustBrainLoss(4)
 	M.hallucination += 10
```

Comment:
These two are the main tradeoff, along with the random movement. Microdosing shouldn't bypass these.